tools/BUILD

```diff
@@ -0,0 +1,45 @@
+package(default_visibility = ["//visibility:public"])
+
+load(":qt_toolchain.bzl", "qt_toolchain")
+
+toolchain_type(name = "toolchain_type")
+
+qt_toolchain(
+    name = "qt_linux",
+    # TODO: determine paths in qt_configure.bzl
+    moc_path = "/usr/bin/moc",
+    rcc_path = "/usr/bin/rcc",
+    uic_path = "/usr/bin/uic",
+)
+
+qt_toolchain(
+    name = "qt_windows",
+    # TODO: determine paths in qt_configure.bzl
+    moc_path = "moc",
+    rcc_path = "rcc",
+    uic_path = "uic",
+)
+
+toolchain(
+    name = "qt_linux_toolchain",
+    exec_compatible_with = [
+        "@platforms//os:linux",
+    ],
+    target_compatible_with = [
+        "@platforms//os:linux",
+    ],
+    toolchain = ":qt_linux",
+    toolchain_type = "@com_justbuchanan_rules_qt//tools:toolchain_type",
+)
+
+toolchain(
+    name = "qt_windows_toolchain",
+    exec_compatible_with = [
+        "@platforms//os:windows",
+    ],
+    target_compatible_with = [
+        "@platforms//os:windows",
```

Comment:
You might want to consider if you want to explicitly say that only x64 is supported. For the moment I don't think that it is a restriction and might work with 32 bit but not sure if will be the case when starting with the linking and compilation flags in the toolchain
`"@platforms//cpu:x86_64"`

Reply:
Thanks, I added that in.